packages/design-system/src/components/web-components/preactement/globalStyles.ts

```diff
@@ -0,0 +1,84 @@
+const POLLING_INTERVAL = 10;
+
+let globalSheets: CSSStyleSheet[] | null = null;
+let subscribers: ShadowRoot[] = [];
+
+export function getGlobalStyleSheets() {
+  if (globalSheets === null) {
+    globalSheets = copyGlobalStyleSheets();
+    watchForFutureChanges();
+  }
+
+  return globalSheets;
+}
+
+function copyGlobalStyleSheets(): CSSStyleSheet[] {
+  return Array.from(document.styleSheets).map((x) => {
```

Comment:
Super nit: Can we replace `x` with `sheet` or something slightly more informative?

Reply:
Yeah, that's a good idea. This original function was [from this helpful blog post](https://eisenbergeffect.medium.com/using-global-styles-in-shadow-dom-5b80e802e89d) for reference

Reply:
[c2cdf75](https://github.com/CMSgov/design-system/pull/3346/commits/c2cdf75ad1584d9cbbf29fb796b498e45955a449)